src/ui/GameUI.cpp

```diff
@@ -715,7 +715,7 @@ void createInventoryTooltipFrame(const int player)
 		auto tooltipTextField = tooltipFrame->addField("inventory mouse tooltip header", 1024);
 		tooltipTextField->setText("Nothing");
 		tooltipTextField->setSize(SDL_Rect{ 0, 0, 0, 0 });
-		tooltipTextField->setFont("fonts/pixelmix.ttf#14");
+		tooltipTextField->setFont("fonts/pixelmix.ttf#16");
```

Comment:
Could you revert these font size changes for my HUD stuff - I'll need to redo the spacing/layout and want the old sizes for reference.